dotcom-rendering/.prout.json

```diff
@@ -0,0 +1,12 @@
+{
+	"checkpoints": {
+		"PROD": {
+			"url": "https://www.theguardian.com/uk",
+			"overdue": "30M"
+		},
+		"CODE": {
+			"url": "https://m.code.dev-theguardian.com/uk",
```

Comment:
Incidentally, have you deployed this to CODE? Did you see the hash?

Reply:
> https://m.code.dev-theguardian.com/uk?dcr does indeed show the commit hash corresponding to https://riffraff.gutools.co.uk/deployment/view/59010e8b-f34c-4f4f-9297-47a68aadf2bc

Ah yeah, I see it!

![image](https://github.com/guardian/dotcom-rendering/assets/52038/b154323f-81a3-408c-b932-882c30794d70)